lib/api/backend/graphql/mutation/user/RefreshSession.graphql

```diff
@@ -21,7 +21,7 @@ mutation RefreshSession($secret: RefreshTokenSecret!) {
         ... on CreateSessionOk {
             user {
                 __typename
-                id
+                ...MyUser
```

Comment:
I think we can omit `MyUser` model retrieval during `RefreshSession`, as it usually happens every 30 minutes after `MyUser` is already authorized, so it won't make sense to update it.